lib/tasks/onboarding_emails.rake

```diff
@@ -0,0 +1,22 @@
+namespace :onboarding_emails do
+  desc "Send onboarding emails to private beta users"
+  task :send, %i[organisation_id host] => :environment do |_task, args|
+    organisation_id = args[:organisation_id]
+    host = args[:host]
```

Comment:
Is there a reason why we make the host configurable?
You still need to have rights to run a task but you could potentially target anything which doesn't sound right to me.

Reply:
I suppose it's always configurable but you're right this could use the ENV var we already set. Have changed it.